pro/src/new_components/OffererHasBeenSoftDelete/OffererHasBeenSoftDelete.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import styles from './OffererHasBeenSoftDelete.module.scss'
+
+const OffererHasBeenSoftDelete = (): JSX.Element => {
+  return (
+    <div className={styles['offerer-soft-deleted-warning']}>
+      <div className={styles['offerer-soft-deleted-warning-inner']}>
+        <p>
+          Votre structure a été désactivée. Pour plus d'informations sur la
+          désactivation veuillez contacter notre support.
```

Comment:
Ce wording a été validée par le métier ?
Si j'en crois l'US, il faudrait un message comme ça (sans oublier l'apostrophe français ’) :

```suggestion
          Votre précédente structure a été désactivée. Pour plus d’informations sur la
          désactivation et vos données, veuillez contacter notre support.
```

Reply:
je viens de revalider avec Charlotte, bien vu pour l'apostrophe !